<launch>
    <param name="sensor_name" value="ids_left" />
    <node name="baxter_sequence_calibration_ids" pkg="calibration_routine" type="baxter_sequence_calibration_ids.py" output="screen"/>
</launch>
